Utilities/Text Utility/textUtil.html

```diff
@@ -90,75 +91,78 @@ <h5 class="h5 m2">Headline 5 </h5>
 
                 <div class="divider"></div>
 
-                <h2>Small Text  </h2>
-                <p>This will make your text smaller , you have to just add class <code>txt--sm</code> in your p tag .  </p>
-    
-<div class="showcase">
+                <h2>Small Text </h2>
+                <p>This will make your text smaller , you have to just add class <code>txt--sm</code> in your p tag .
+                </p>
 
-    <p class="txt--sm">This is a small Text . </p>
-</div>
-<div class="container">
-    <div class="code-wrapper">
-        <pre>
+                <div class="showcase">
+
+                    <p class="txt--sm">This is a small Text . </p>
+                </div>
+                <div class="container">
+                    <div class="code-wrapper">
+                        <pre>
 
     <code class="code">
 &lt;p class="txt--sm">This is a small Text . &lt;/p></code>
   </pre>
-        <button id="copy-btn">Copy</button>
-    </div>
+                        <button id="copy-btn">Copy</button>
+                    </div>
 
 
-    <div class="divider"></div>
+                    <div class="divider"></div>
 
 
```

Comment:
remove extra blank spaces please

Reply:
I have used beautify to format the code .